mmaction/datasets/pipelines/formating.py

```diff
@@ -201,12 +236,16 @@ def __call__(self, results):
             for key in self.meta_keys:
                 meta[key] = results[key]
             data[self.meta_name] = DC(meta, cpu_only=True)
+        if self.nested:
```

Comment:
What problem does the introduced `nested` solve?

Reply:
To be compatible with mmdet testing:
In mmdet, during testing, the input is list of imgs and img_metas.

Reply:
ok